docs/lib/graphqlapi/fragments/flutter/authz/30_multi.md

```diff
@@ -1,7 +1,9 @@
-<amplify-callout>
+When you have configured multiple APIs, you can specify the name of the API as a parameter as the target for an operation:
 
-Using multiple modes of authorization at once is not currently supported by Flutter. We are actively working on this. 
-
-We have created a [Github Issue](https://github.com/aws-amplify/amplify-flutter/issues/308) to track this missing feature. 
-
-</amplify-callout>
\ No newline at end of file
+```dart
+var operation = Amplify.API.mutate<String>(
+    request: GraphQLRequest(
+        document: graphQLDocumentString, apiName: '[FRIENDLY-NAME-API-WITH-API-KEY]'));
```

Comment:
nit: could place `apiName` on a different row for readability on narrow screens

Reply:
done, good call